### ISSUE TITLE
GitHub: Invite member 

### ISSUE THREAD

**Issue by mhellmeier** [eclipse-tractusx/sig-infra#414]
Who should be invited to [Catena-X ng](https://github.com/catenax-ng)

GitHub User: @eckardg

Vouching person: Me (@mhellmeier)

Please also add @eckardg to the `traceability-spec` team.

**Comment by FaGru3n** [comment]
Hi @mhellmeier could you please verify that @eckardg will working for Catena-X?

**Comment by mhellmeier** [comment]
Yes, I can verify that the person behind the account @eckardg is working for Catena-X.

**Comment by FaGru3n** [comment]
Hi @mhellmeier thanks and invitation to @eckardg was sent.

![image](https://github.com/eclipse-tractusx/sig-infra/assets/121097161/f4d1ba48-8814-4981-930d-a1fe15a940fd)

for [traceability-spec](https://github.com/orgs/catenax-ng/teams/traceability-spec) you as maintainer can also add him to the team as well.

Mabye also a good option to add more maintainer to the gh-team as well 
https://github.com/orgs/catenax-ng/teams/traceability-spec?query=role%3Amaintainer

For now @eckardg has to accept the invite and then can be added to the gh-team (7 days)
